### PR TITLE
 add support for Node.js 12 on UBI8

### PR DESCRIPTION
**Pull request by danbev**
This commit adds the rpmdevtools rpms to this repository as I've failed
to find them in a repository for RHEL8 anywhere. Docker will copy these
and then install them.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+COPY rpmdevtools/ /root/rpmdevtools/
+RUN pushd /root/rpmdevtools && yum -y --nogpgcheck localinstall *.rpm && popd
+RUN yum install -y git                            \
+                   gcc                            \
+                   gcc-c++                        \
+                   openssl-devel                  \
+                   libicu-devel                   \
+                   procps-ng                      \
+                   python2-devel                  \
+                   python3-devel                  \
```
Comment (lance) [inline Dockerfile:12]: Why both Python 2 and 3? I haven't built Node 12 yet. Has it moved to Python 3?

**Review by lance — APPROVED** [review]
Just one question. Otherwise LGTM.

File: Dockerfile (hunk 1 above)
Comment (danbev) [reply Dockerfile:12]: There is one script in V8 which I believe needs Python3. I'm not 100% about the details as this was something that Yaakov provided in his work. The goal is to migrate all the scrips to Python3 eventually so I think we'll have to live with Python2 for a while still.

Comment (helio-frota) [reply Dockerfile:12]: Extra information about python 2x support:
https://github.com/python/devguide/pull/344

**Review by helio-frota — APPROVED** [review]

**Comment by danbev** [comment]
Landed in df7b288, and 3f9b6d0.

**Pull request closed without merging**